packages/fast-documentation/package.json

```diff
@@ -0,0 +1,130 @@
+      "/.tmp/*"
+    ],
+    "testURL": "http://localhost",
+    "transform": {
+      "^.+\\.tsx?$": "ts-jest",
+      "^.+\\.jsx?$": "babel-jest"
+    },
+    "transformIgnorePatterns": [
+      "!<rootDir>/node_modules/lodash-es",
+      "node_modules/jss-*",
+      "node_modules/css-vendor"
+    ],
+    "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$",
+    "moduleFileExtensions": [
+      "ts",
+      "tsx",
+      "js",
+      "jsx",
+      "json"
+    ],
+    "moduleDirectories": [
+      "node_modules",
+      "src"
+    ]
+  },
+  "devDependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-react": "^7.0.0",
+    "@microsoft/fast-tooling-react": "^1.32.0",
+    "@microsoft/fast-tslint-rules": "^3.0.4",
+    "@types/jest": "^24.0.11",
+    "@types/lodash-es": "^4.17.0",
+    "@types/node": "^9.4.6",
+    "@types/react": "^16.4.18",
+    "@types/react-router": "^5.0.3",
+    "@types/react-router-dom": "^4.3.4",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^24.5.0",
+    "babel-loader": "^8.0.6",
+    "enzyme": "^3.3.0",
+    "enzyme-adapter-react-16": "^1.1.1",
+    "favicons-webpack-plugin": "0.0.9",
+    "file-loader": "^1.1.11",
+    "focus-visible": "^4.1.5",
+    "html-webpack-plugin": "^3.0.4",
+    "jest": "^24.5.0",
+    "lint-staged": "^8.1.0",
+    "lodash-es": "^4.0.0",
+    "prettier": "1.18.2",
+    "react": "^16.6.0",
+    "react-docgen-typescript-loader": "^3.1.0",
+    "react-dom": "^16.6.0",
+    "react-router-dom": "^5.0.1",
+    "rimraf": "^2.6.2",
+    "ts-jest": "^24.0.0",
+    "ts-loader": "^4.0.1",
+    "tslint": "^5.9.1",
+    "tslint-config-prettier": "^1.15.0",
+    "tslint-loader": "^3.6.0",
+    "typescript": "^3.3.3",
+    "webpack": "^4.1.0",
+    "webpack-bundle-analyzer": "^2.13.1",
+    "webpack-cli": "^3.1.1",
+    "webpack-dev-server": "^3.1.0",
+    "webpack-shell-plugin": "^0.5.0",
+    "yargs": "^11.0.0"
+  },
+  "dependencies": {
+    "@microsoft/fast-colors": "5.0.2",
+    "@microsoft/fast-components-class-name-contracts-base": "^4.6.0",
+    "@microsoft/fast-components-class-name-contracts-msft": "^4.8.0",
+    "@microsoft/fast-components-foundation-react": "^3.1.3",
+    "@microsoft/fast-components-react-base": "^4.24.1",
+    "@microsoft/fast-components-react-msft": "^4.29.3",
+    "@microsoft/fast-components-styles-msft": "^4.28.1",
+    "@microsoft/fast-glyphs-msft": "^4.0.4",
+    "@microsoft/fast-jss-manager-react": "^4.6.1",
+    "@microsoft/fast-jss-utilities": "^4.7.5",
+    "@microsoft/fast-layouts-react": "^4.11.1",
+    "@microsoft/fast-web-utilities": "^4.4.0",
+    "exenv-es6": "^1.0.0",
+    "lodash-es": "^4.0.0",
+    "react": "^16.6.0",
+    "react-dom": "^16.6.0",
+    "utility-types": "^2.1.0"
+  },
+  "lint-staged": {
+    "*.{ts,tsx}": [
+      "prettier --write",
+      "git add"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  }
```

Comment:
I believe this is only required at the root, where it already exists. this can be removed along with the dependencies above.